python/mlx/nn/layers/upsample.py

```diff
@@ -154,6 +218,18 @@ class Upsample(Module):
                [1.66667, 2, 2.33333, 2.66667],
                [2.33333, 2.66667, 3, 3.33333],
                [3, 3.33333, 3.66667, 4]], dtype=float32)
+        >>> b = nn.Upsample(scale_factor=2, mode='cubic')
```

Comment:
I would remove the extra examples and add the new mode in the docs above.

Reply:
nice catch. I forgot to update the docstring